usecases/monitoring/prometheus.go

```diff
@@ -714,6 +720,29 @@ func newPrometheusMetrics() *PrometheusMetrics {
 			Help:    "Number of requests required to process an entire (user) batch",
 			Buckets: []float64{1, 2, 5, 10, 100, 1000},
 		}, []string{"vectorizer"}),
+		TokenizerDuration: promauto.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "tokenizer_duration_microseconds",
```

Comment:
why `microseconds`? The usual practice is to use `_seconds` and specify appropriate buckets for your data resolution